README.md

```diff
@@ -26,15 +26,15 @@ For more detailed information about `mender-convert` please visit the
 Download the raw Raspberry Pi disk image into a subdirectory input:
 
 ```bash
-mkdir -p input
-cd input
-wget https://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-06-24/2019-06-20-raspbian-buster-lite.zip
+$ mkdir -p input
+$ cd input
+$ wget https://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-06-24/2019-06-20-raspbian-buster-lite.zip
```

Comment:
The intention of not having the `$` is to make it more copy/paste friendly. If the `$` is included, then you need to remove this character when copying the line.

For this reason I would not include this.